bin/dynamic_config/set_hoc_flags.rb

```diff
@@ -35,24 +35,24 @@ def main(argv)
   case mode
     when 'red'
       Gatekeeper.set('postMilestone', value: false)
-      Gatekeeper.set('hoc_activity_writes_disabled', value: true)
       Gatekeeper.set('async_activity_writes', value: true)
       Gatekeeper.set('tracking_pixel_enabled', value: false)
       Gatekeeper.set('puzzle_rating', value: false)
+      DCDO.set('hoc_activity_sample_weight', 0)  # Don't write to hoc_activity.
```

Comment:
In retrospect, I think this should be some large number (some small percentage) rather than 0 (no percentage). We can certainly handle a small number of writes... And it would be a travesty to turn off data collection entirely.

Does making this 100 or 500 or even 1000 seem practical? My suggestion would be to make this change with another deploy. And if we do turn to red, update the DCDO setting manually after flipping to red.

What do you think?